.claude/commands/commentreply.py

```
@@ -0,0 +1,715 @@
+    success_issue, issue_data, _ = run_command(issue_cmd, "fetch issue comments", timeout=120)
+
+    if not success_review or not success_issue:
+        print("❌ ERROR: Failed to fetch fresh comments from GitHub API")
+        if not os.path.exists(output_file):
+            sys.exit(1)
+        print("⚠️  Falling back to stale cache...")
+        return
+
+    # Parse and combine comments
+    try:
+        review_comments = [json.loads(l) for l in (review_data or "").splitlines() if l.strip()]
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)